From 8e895c79aa0dd8c10c9006883d5f45269fc8e1f3 Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Fri, 2 Aug 2019 08:22:03 +0200
Subject: [PATCH 28/38] account: Make the avatar chooser fit the phone

This makes the avatar chooser popover better fit a small and narrow
phone screen.
---
 .../pages/account/gis-account-avatar-chooser.ui    | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

--- a/gnome-initial-setup/pages/account/gis-account-avatar-chooser.ui
+++ b/gnome-initial-setup/pages/account/gis-account-avatar-chooser.ui
@@ -2,8 +2,8 @@
 <interface>
   <!-- interface-requires gtk+ 3.8 -->
   <template class="UmPhotoDialog" parent="GtkPopover">
-    <property name="height-request">360</property>
-    <property name="width-request">480</property>
+    <property name="height-request">480</property>
+    <property name="width-request">360</property>
     <child>
       <object class="GtkBox">
         <property name="visible">True</property>
@@ -12,18 +12,24 @@
           <object class="GtkFlowBox" id="recent_pictures">
             <property name="visible">True</property>
             <property name="halign">start</property>
-            <property name="margin">20</property>
+            <property name="margin">8</property>
             <property name="margin-bottom">0</property>
             <property name="selection-mode">none</property>
           </object>
         </child>
         <child>
+          <object class="GtkScrolledWindow">
+            <property name="visible">True</property>
+            <property name="vexpand">True</property>
+        <child>
           <object class="GtkFlowBox" id="flowbox">
             <property name="visible">True</property>
-            <property name="border-width">20</property>
+            <property name="border-width">8</property>
             <property name="selection-mode">none</property>
           </object>
         </child>
+          </object>
+        </child>
         <child>
           <object class="GtkBox">
             <property name="visible">True</property>
